docs/pubsub.rst

```diff
@@ -460,20 +460,136 @@ You can add custom transport using the attribute ``rtps.userTransports``.
+|     //Link the Transport Layer to the Participant                               |             </transport>                                            |
+|     Pparams.userTransports.push_back(my_transport);                             |         </transports>                                               |
+|                                                                                 |                                                                     |
+|                                                                                 |         <participant profile_name="my_transport">                   |
+|                                                                                 |             <rtps>                                                  |
+|                                                                                 |                 <userTransports>                                    |
+|                                                                                 |                     <transport_id>my_transport</transport_id>       |
+|                                                                                 |                 </userTransports>                                   |
+|                                                                                 |                 <useBuiltinTransports>false</useBuiltinTransports>  |
+|                                                                                 |             </rtps>                                                 |
+|                                                                                 |         </participant>                                              |
+|                                                                                 |         ...                                                         |
+|                                                                                 |     </profiles>                                                     |
++---------------------------------------------------------------------------------+---------------------------------------------------------------------+
+
+
+TCP Transport
+"""""""""""""
+
+To use TCP transports you need to define some more configurations:
+
+You must create a new TCP transport descriptor, for example TCPv4.
+This transport descriptor has a field named ``listening_ports`` that indicates to Fast-RTPS in which physical TCP
+port our participant will listen for input conections. If ommited, the participant will not be able to receive incoming
+connections, but will be able to connect to others participants that have configured their listening ports.
+The transport must be added in the ``userTransports`` list of the participant attributes.
+
+To configure the participant to connect to another node through TCP, you must configure add Locator to its ``initialPeersList`` that points to the remote *listening port*.
```

Comment:
Divide next table in two. One configuration the participant that is the server (using the listening port), and other table with the configuration of the participant that is the client (setting the initial peers).